packages/keystone/KSv5v6/v5/registerSchema.js

```diff
@@ -33,28 +56,34 @@ function applyKeystoneV5AdminFixes (schema) {
 /** @deprecated it's part of internal API use prepareKeystone instead */
 function convertStringToTypes (schema) {
     const mapping = {
+        // Default types
+        AuthedRelationship,
         CalendarDay,
         Checkbox,
         DateTimeUtc,
         Decimal,
-        File: FileWithUTF8Name,
         Float,
         Integer,
+        Markdown,
         Password,
         Relationship,
-        Select,
         Slug,
-        Text,
         Url,
         Uuid,
         Virtual,
-        Json,
-        SignedDecimal,
+        Wysiwyg,
+
+        // Custom types
+        AddressPartWithType,
         AutoIncrementInteger,
-        LocalizedText,
-        AuthedRelationship,
-        HiddenRelationship,
         DateInterval,
+        File: FileWithUTF8Name,
+        Json,
+        HiddenRelationship,
+        LocalizedText,
+        Select,
+        SignedDecimal,
+        Text,
```

Comment:
Should we throw a error now in non-string cases few lines below?
<img width="923" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/c9d3db9f-0c13-4f5f-8c5a-c46c2a2f3ba2">

Reply:
added